jina/optimizers/__init__.py

```diff
@@ -38,17 +37,17 @@ def __call__(self, response):
 class OptunaOptimizer:
     def __init__(
         self,
-        index_flow_runner,
-        eval_flow_runner,
+        multi_flow,
```

Comment:
is it necessarily a multiFlow?

Reply:
It can also be just one flow but has to be wrapped in `MultiFlowRunner`

Reply:
that's why I think it would be good to have `MultiFlowRunner`  inherit from `FlowRunner`. But no problem